<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-8UCFDRT" ignored="0" name="TestNGTest1" tests="3" failures="1" timestamp="2020-08-14T17:25:23 EDT" time="180.721" errors="0">
  <testcase name="SignUpTest" time="22.513" classname="com.testCases.TC_Android_001"/>
  <testcase name="SignUpTest" time="18.043" classname="com.testCases.TC_Android_001">
    <failure type="java.lang.AssertionError" message="expected [please enter a valid email] but found [Purposely fail:please enter a valid email]">
      <![CDATA[java.lang.AssertionError: expected [please enter a valid email] but found [Purposely fail:please enter a valid email]
at com.utilities.CheckPoint.validationResult(CheckPoint.java:16)
at com.testCases.TC_Android_001.SignUpTest(TC_Android_001.java:74)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- SignUpTest -->
  <testcase name="SignUpTest" time="18.597" classname="com.testCases.TC_Android_001"/>
</testsuite> <!-- TestNGTest1 -->
